templates/transcriptAnalyzer/TNBDE.html

```diff
@@ -95,6 +95,19 @@
         <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
 		<script src="{% static 'transcriptAnalyzer/TNBDE.js' %}"></script>
 		<script src="{% static 'transcriptAnalyzer/sorttable.js' %}"></script>
-		
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.2.8/ace.js"></script>
+		<script>
+			ace.config.set("packaged", true);
+			ace.config.set("basePath", "https://cdnjs.cloudflare.com/ajax/libs/ace/1.2.8/ace.js");
+			var sqlEditor=ace.edit("sql");

```

Comment:
Any reason this script block can't be inserted into the document-ready part in `TNBDE.js`?

Reply:
ACE requires somethings to run before document.onload and some after so I am not sure if it would work there but I can check